packages/components/addon/components/hds/app-footer/index.hbs

```diff
@@ -0,0 +1,13 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<div class={{this.classNames}} ...attributes>
+  {{! topContent structure is temporary, will be refactored when final design is ready }}
+  <div class="hds-app-footer__top-content">{{yield to="topContent"}}</div>
```

Comment:
I will probably refactor the DOM structure & layout styles if we keep this topContent yield in the final design.

Reply:
Not sure if you want to use named blocks for the child elements of the `AppFooter`, but in case there may be downsides in not using just yielded components (I can explain more, if/when needed).

Reply:
I think @andgen404 may be changing this layout so I'll wait to see what the final design is then I can refactor. The structure I have in there now is mainly to demonstrate the layout.

Reply:
After refactoring to implement the latest designs I took out named blocks.